src/screens/Collectible/Collectible.js

```diff
@@ -111,6 +115,12 @@ class CollectibleScreen extends React.Component<Props> {
     this.props.navigation.navigate(SEND_COLLECTIBLE_FROM_ASSET_FLOW, { assetData });
   };
 
+  onTouchImage = () => {
+    const { navigation: { state: { params: { assetData } } } } = this.props;
+
+    this.props.navigation.navigate(FULL_SCREEN_COLLECTIBLE, { collectible: assetData });
```

Comment:
```suggestion
    navigate(FULL_SCREEN_COLLECTIBLE, { collectible: assetData });
```

Reply:
Damn, they need to add multi-line suggestions :sweat_smile: